keyboards/nifty_numpad/keymaps/idle_rgb_example/keymap.c

```diff
@@ -0,0 +1,177 @@
+// Copyright 2023 Isaac Rex (@Acliad)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+// Layers 
+// NOTE: LAYER_RGB is defined in nifty_numpad.h
+enum LAYERS {
+    LAYER_BL = 0,
+    LAYER_MOD,
+};
+
+// Layer Indicator LED index
+#define _NUM_LED_INDEX 8
+
+#define TAPPING_TERM 200
```

Comment:
That's not how this works.  This needs to go into a config.h file.

Reply:
Ah, I see. I have added a config.h to this keymap and moved TAPPING_TERM to that file. I assume _NUM_LED_INDEX can remain in the .c, or would it be preferable to move that to a header file? Thanks!

Reply:
If it's only being used in that file, then yeah, it can stay. 

But in general, global settings, like tapping term need to go in config.h or the like.